app/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -318,4 +324,30 @@ describe("Summary", () => {
       expect(errorContainer).toHaveTextContent(errorMessage);
     });
   });
+
+  it("handles dirty Live to Review as expected", async () => {
+    const { mockRollout, rollout } = mockLiveRolloutQuery("demo-slug", {
+      status: NimbusExperimentStatusEnum.LIVE,
+      publishStatus: NimbusExperimentPublishStatusEnum.DIRTY,
+      statusNext: null,
+    });
+
+    const mutationMock = createMutationMock(
+      rollout.id!,
+      NimbusExperimentPublishStatusEnum.DIRTY,
+      {
+        changelogMessage: CHANGELOG_MESSAGES.REQUESTED_REVIEW_UPDATE,
+        statusNext: null,
+        publishStatus: NimbusExperimentPublishStatusEnum.DIRTY,
+        status: NimbusExperimentStatusEnum.LIVE,
+      },
+    );
+    render(<Subject props={rollout} mocks={[mockRollout, mutationMock]} />);
+
+    const requestUpdateButton = await screen.findByTestId(
```

Comment:
can you also add a test that end enrollments and other buttons are still visible with this? :)

Reply:
one thing: I think it's worth saying, instead of End experiment-> End rollout, what do you think?

Reply:
Ooo yeah we can definitely do that. I'll file a ticket!

Reply:
https://github.com/mozilla/experimenter/issues/8368

Reply:
fantastic, thank you 🙏